### PR TITLE
WLT-727: edited getBalance method

### PR DESCRIPTION
**Pull request by pol9kov**
**- What I did**
Added deposits to getBalance method like in openAPI
**- How I did it**
Changed getBalance method; changed deposit marshal to map; add struc GetBalanceResponse. 
**- How to verify it**
We can test getBalance deposits only in migration tests when migration will be done
**- Description for the changelog**

GetBalance method update

### CODE REVIEW

File: logicrunner/builtin/contract/member/member.go [hunk 1]
```diff
@@ -458,50 +465,67 @@ func (m *Member) migration(txHash string, burnAddress string, amount big.Int, un
...
-// FindDeposit finds deposits for this member with this transaction hash.
-func (m *Member) FindDeposit(txHash string, inputAmountStr string) (bool, deposit.Deposit, error) {
+// GetDeposits get all deposits for this member
+func (m *Member) GetDeposits() ([]map[string]string, error) {
+	var result []map[string]string
+	for _, dRef := range m.Deposits {
+
+		d := deposit.GetObject(dRef)
+
+		depositInfo, err := d.MapMarshal()
```
Comment (LinnikD) [inline logicrunner/builtin/contract/member/member.go]: Its better to use smth like DepositResoinse object with tags for json marshaling in GetBalanceResponse

File: logicrunner/builtin/contract/deposit/deposit.go [hunk 2]
```diff
@@ -75,14 +75,8 @@ func New(migrationDaemonConfirms map[insolar.Reference]bool, txHash string, amou
...
 // MapMarshal gets deposit information.
-func (d *Deposit) MapMarshal() (map[string]string, error) {
-	return map[string]string{
-		"timestamp":       d.Timestamp.String(),
-		"holdReleaseDate": d.HoldReleaseDate.String(),
-		"amount":          d.Amount,
-		"bonus":           d.Bonus,
-		"txId":            d.TxHash,
-	}, nil
+func (d *Deposit) Itself() (Deposit, error) {
```
Comment (LinnikD) [inline logicrunner/builtin/contract/deposit/deposit.go]: Why not `Get`?

Comment (pol9kov) [reply logicrunner/builtin/contract/deposit/deposit.go]: Because get what?

Comment (pol9kov) [reply logicrunner/builtin/contract/deposit/deposit.go]: Mby GetItself? But we can avoid get I think.

Comment (pol9kov) [reply logicrunner/builtin/contract/deposit/deposit.go]: But I thing Itself() is better

**Pull request closed without merging**